### PR TITLE
Restrictions to commands players can send on Geyser Standalone

### PR DESCRIPTION
**Pull request by Konicai**
- Players can no longer create dumps on standalone
- If the geyser version command is sent by a player on standalone, it will not check for new updates
- If the geyser stop command is sent by a player on standalone, they are now told they don't have permission

Resolves issue #2097

### CODE REVIEW

File: connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java [hunk 1]
```diff
@@ -62,28 +67,31 @@ public void execute(GeyserSession session, CommandSender sender, String[] args)
...
 
         // Disable update checking in dev mode
+        // Disable update checking for players in Geyser Standalone
         //noinspection ConstantConditions - changes in production
         if (!GeyserConnector.VERSION.equals("DEV")) {
-            sender.sendMessage(LanguageUtils.getPlayerLocaleString("geyser.commands.version.checking", sender.getLocale()));
-            try {
-                Properties gitProp = new Properties();
-                gitProp.load(FileUtils.getResource("git.properties"));
+            if (!(!sender.isConsole() && connector.getPlatformType() == PlatformType.STANDALONE)) {
```
Comment (Camotoy) [inline connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java:73]: You can add this to the above `if` statement check.

File: connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java [hunk 2]
```diff
@@ -62,28 +67,31 @@ public void execute(GeyserSession session, CommandSender sender, String[] args)
         sender.sendMessage(LanguageUtils.getPlayerLocaleString("geyser.commands.version.version", sender.getLocale(), GeyserConnector.NAME, GeyserConnector.VERSION, GeyserConnector.MINECRAFT_VERSION, bedrockVersions));
 
         // Disable update checking in dev mode
+        // Disable update checking for players in Geyser Standalone
```
Comment (Camotoy) [inline connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java:70]: Put this in the same sentence as the above comment.

File: connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java (hunk 1 above)
Comment (Konicai) [reply connector/src/main/java/org/geysermc/connector/command/defaults/VersionCommand.java:73]: Will do, wasn't sure if it was getting too messy

**Review by Camotoy — APPROVED** [review]

**Comment by Camotoy** [comment]
Thanks!

**Comment by Konicai** [comment]
❤️

**Comment by ShadowFlare** [comment]
Wow, thanks for implementing this already. Already got it up and running on my server and seems good.

**Comment by Konicai** [comment]
Awesome!

**Pull request merged**